include/LMS1xx/colaa.h

```diff
@@ -0,0 +1,268 @@
+#ifndef COLAA_H
```

Comment:
Jup. True. But the you at least need the minimal copyright/license statement. Without, you can never be sure what the file is actually licensed. 

So please add.